osmosis-1/osmosis.zone_assets.json

```diff
@@ -1517,6 +1517,16 @@
       "path": "transfer/channel-874/untrn",
       "osmosis_main": false,
       "osmosis_frontier": true
+    },
+    {
+      "chain_name": "juno",
+      "base_denom": "cw20:juno1ju8k8sqwsqu5k6umrypmtyqu2wqcpnrkf4w4mntvl0javt4nma7s8lzgss",
+      "path": "transfer/channel-169/cw20:juno1ju8k8sqwsqu5k6umrypmtyqu2wqcpnrkf4w4mntvl0javt4nma7s8lzgss",
+      "osmosis_main": false,
+      "osmosis_frontier": true,
+      "override_properties": {
```

Comment:
No need for this override since the asset is already registered as CASA in the Chain Registry

Reply:
Thank you. We're trying to update the Casa logo so the pool (1028) adds the logo + APR for the rewards we deposited. I appreciate your help

Reply:
Right, we still are going to add the asset, but the `override_properties` property should be removed since it's not changing anything.

Reply:
Thank you. So in this case we just remove everything in line 1527 correct?

Reply:
yes, please remove what you have 1527-1529